tests/posix/stand/pty_uni.std

```diff
@@ -17,102 +17,98 @@ pty gives me: "hello, world"
 "   UDB Version "
 "   UDB Version 2"
 "   UDB Version 2."
-"   UDB Version 2.0"
```

Comment:
We should find away to ignore this part to avoid having to update the test output every time UDB version changes, although that rarely happens.